src/jit/codegeninterface.h

```diff
@@ -98,7 +98,13 @@ class CodeGenInterface
     bool      m_genAlignLoops;
 
 private:
+#if defined(_TARGET_XARCH_)
+    static const insFlags instInfo[INS_count];
+#elif defined(_TARGET_ARM_) || defined(_TARGET_ARM64_)
     static const BYTE instInfo[INS_count];
```

Comment:
Ideally, `arm` and `arm64` would also port their individual 'flags' parameters to also be a single combined 'flags' parameter.